components/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt/src/main/java/org/wso2/carbon/identity/api/resource/mgt/constant/APIResourceManagementConstants.java

```diff
@@ -142,7 +142,8 @@ public enum ErrorMessages {
                 "properties.", "Error while adding API resource properties to the database."),
         ERROR_CODE_ERROR_WHILE_UPDATING_SCOPE_METADATA("65015", "Error while updating scope metadata.",
                 "Error while updating scope metadata in the database."),
-        ;
+        ERROR_CODE_ERROR_WHILE_RESOLVING_TENANT_FOR_ORGANIZATION("65016", "Error while resolving tenant",
```

Comment:
Wouldn't it be better to log the tenant domain/ organization id with this error so that debugging errors would be easier?

Reply:
Added the tenant domain.